shared_sql_utils/src/dsn.rs

```diff
@@ -228,14 +228,16 @@ impl Dsn {
     }
 
     pub fn to_connection_string(&self) -> String {
-        self.iter()
-            .map(|(key, value)| {
-                if value.is_empty() {
-                    return "".into();
-                }
-                format!("{key}={value};")
-            })
-            .collect::<String>()
```

Comment:
This was a clever catch on clippy's part. I followed their example [here](https://rust-lang.github.io/rust-clippy/master/index.html#/format_collect) to fix it.